.spelling

```diff
@@ -149,6 +150,7 @@ Reviewer2
 SNI
 SREs
 Secura
+ServiceEntry
```

Comment:
Don't want in the dictionary. In the text, either put backticks around ServiceEntry, or simply expand to 'service entry' as appropriate.